rollup.config.js

```diff
@@ -0,0 +1,27 @@
+import resolve from 'rollup-plugin-node-resolve';
+import commonjs from 'rollup-plugin-commonjs';
+import uglify from 'rollup-plugin-uglify';
+import yargs from 'yargs';
+
+const args = yargs.argv;
+
+export default {
+  input: './lib/index.js',
+  output: {
+    file: `dist/index${args.min ? '.min' : ''}.js`,
+    format: 'umd',
+  },
+  name: 'ReactBeautifulDnd',
+  plugins: [
+    resolve({
+      browser: true,
```

Comment:
why is this needed?

```js
// some package.json files have a `browser` field which
      // specifies alternative files to load for people bundling
      // for the browser. If that's you, use this option, otherwise
      // pkg.browser will be ignored
      browser: true,  // Default: false
```